src/main/java/frc/robot/subsystems/Storage.java

```diff
@@ -3,14 +3,22 @@
 import com.revrobotics.CANSparkMax;
 import com.revrobotics.CANSparkBase.IdleMode;
 import com.revrobotics.CANSparkLowLevel.MotorType;
+import edu.wpi.first.wpilibj.DigitalInput;
 
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 import frc.robot.constants.StorageConstants;
 
 public class Storage extends SubsystemBase {
 
-    private final CANSparkMax storageMotor = new CANSparkMax(StorageConstants.motorPort,
+    private final CANSparkMax transitionMotor = new CANSparkMax(StorageConstants.transitionMotorPort,
             MotorType.kBrushless);
+    private final CANSparkMax leftStorage = new CANSparkMax(
+            StorageConstants.leftStoragePort, MotorType.kBrushless);
+    private final CANSparkMax rightStorage = new CANSparkMax(
+            StorageConstants.rightStoragePort, MotorType.kBrushless);
+
+    private final DigitalInput beginLaser = new DigitalInput(StorageConstants.beginLaserPort);
```

Comment:
I think these variables can have better names than "beginLaser" and "endLaser." I think you should name the variables based on the locations of the sensors on the robot.

Reply:
I agree